[rospy.client][INFO] 2021-10-22 12:30:51,968: init_node, name[/ss42/decoder_node], pid[19149]
[xmlrpc][INFO] 2021-10-22 12:30:51,971: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2021-10-22 12:30:51,974: ROS Slave URI: [http://ss42.local:41589/]
[xmlrpc][INFO] 2021-10-22 12:30:51,973: Started XML-RPC server [http://ss42.local:41589/]
[rospy.impl.masterslave][INFO] 2021-10-22 12:30:51,976: _ready: http://ss42.local:41589/
[xmlrpc][INFO] 2021-10-22 12:30:51,978: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-10-22 12:30:51,981: Registering with master node http://ss42.local:11311/
[rospy.init][INFO] 2021-10-22 12:30:52,075: registered with master
[rospy.rosout][INFO] 2021-10-22 12:30:52,076: initializing /rosout core topic
[rospy.rosout][INFO] 2021-10-22 12:30:52,096: connected to core topic /rosout
[rospy.simtime][INFO] 2021-10-22 12:30:52,119: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-10-22 12:30:52,183: [/ss42/decoder_node] ~publish_freq = 2.0 
[rospy.internal][INFO] 2021-10-22 12:30:52,405: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-10-22 12:30:54,048: topic[/ss42/camera_node/image/raw] adding connection to [/ss42/raw_cam_info_reader_node], count 0
[rospy.internal][INFO] 2021-10-22 12:30:54,519: topic[/ss42/camera_node/image/compressed] adding connection to [http://ss42.local:42529/], count 0
[rospy.internal][INFO] 2021-10-22 12:32:06,867: topic[/ss42/camera_node/image/raw] adding connection to [/ss42/ground_projection], count 1
[rospy.internal][INFO] 2021-10-22 12:32:07,569: topic[/ss42/camera_node/image/raw] removing connection to /ss42/ground_projection
[rospy.core][INFO] 2021-10-22 12:33:55,335: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2021-10-22 12:33:55,403: Unable to initiate TCP/IP socket to ss42.local:33593 (http://ss42.local:42529/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-10-22 12:33:55,410: topic[/ss42/camera_node/image/raw] removing connection to /ss42/raw_cam_info_reader_node
[rospy.internal][INFO] 2021-10-22 12:33:55,414: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-10-22 12:33:55,418: topic[/ss42/camera_node/image/compressed] removing connection to http://ss42.local:42529/
[rospy.impl.masterslave][INFO] 2021-10-22 12:33:55,418: signal-2
[rospy.core][INFO] 2021-10-22 12:33:55,983: signal_shutdown [atexit]
